script.js

```diff
@@ -409,26 +428,15 @@ function handleData([
  *****************************************/
 
 function handleCitiesLayer(geojson) {
-  // styling for the cities layer: style cities conditionally according to a presence of a moratorium
+  // styling for the cities layer: style cities conditionally according to moratorium rating scale 1 to 3
   const pointToLayer = (feature, latlng) => {
-    // style cities based on whether their moratorium has passed
-    if (feature.properties.passed === "Yes") {
-      return L.circleMarker(latlng, {
-        color: "#4dac26",
-        fillColor: "#b8e186",
-        fillOpacity: fillOpacity,
-        radius: pointRadius,
-        weight: strokeWeight
-      });
-    } else {
-      return L.circleMarker(latlng, {
-        color: "#d01c8b",
-        fillColor: "#f1b6da",
-        fillOpacity: fillOpacity,
-        radius: pointRadius,
-        weight: strokeWeight
-      });
-    }
+    return L.circleMarker(latlng, {
+      color: feature.properties.range?strokeColorScale[feature.properties.range]:"#808080",
```

Comment:
@jimboweb the application of the color array could be simplified to just an 'or':

```js
color: strokeColorScale[feature.properties.range] || "#808080",
```

Probably worth setting a variable for the fallback color instead of using the string as well. E.g. at the top: 

```js
const colorNoData = "#808080";
```

Then when using it:

```js
color: strokeColorScale[feature.properties.range] || colorNoData,
```